### ISSUE TITLE
Startup And Uninstall Docker Desktop for Windows Failing

### ISSUE THREAD

**Issue by KingsleyElder** [docker/for-win#5616]
### Expected behavior
Docker Desktop will successfully start up and uninstalling Docker from "Apps and Features" should remove the installation

### Actual behavior
Docker Desktop does not complete "starting up", crashes and does not allow restart

### Information
Uploaded two diagnostics payloads
- Diagnostics ID:  77264CE3-1444-46CE-A6B9-C45284F37B50/20200127230535
- Diagnostics ID:  F4AC0BE5-64E6-4369-B81C-C433AF06EB17/20200127233357

**Is it reproducible?**
In the machines current state it is reproducible, but I'm not sure what caused the current state of the installation that prevents docker starting up

**Is the problem new?**
Since this is a new installation with my account on this Windows machine, it is new
Downloaded from https://docs.docker.com/docker-for-windows/

  - Windows Version: 10 Enterprise
  - Docker Desktop Version: 19.03.5, build 633a0ea
  - Running on physical machine with 32GB RAM

### Steps taken to troubleshoot 
- I've reviewed this post that provided a PS script: https://forums.docker.com/t/cannot-uninstall-docker-for-windows/15661
- Launching the uninstaller from from "Apps and Features", the uninstall never completed, though the script mentioned in the article must have been removed since it was not available in the expected location: C:\Program Files\Docker\Docker\resources\MobyLinux.ps1
- Running the installer again did not repair the installation but the message suggested the Docker installation was in order
- I had to use CCleaner to remove registry entries before I was abler to re-run the installer
- As with the initial installation, the docker icon in the tray said "Docker Desktop starting up..." but eventually crashed
- Restarting from the icon repeatedly failed

**Comment by PaulGilchrist** [comment]
I have this same issue

**Comment by brianmaher84** [comment]
Have you tried following steps I used to recover:
https://github.com/docker/for-win/issues/5570

**Comment by KingsleyElder** [comment]
The resolution for me was to manually delete files and folders then install the previous stable version. It could have just been the state of my machine or that I had recently picked up all the recent Windows updates when trying to install Docker 19.03.5.

I resorted to the manual uninstall since I could not uninstall it from Programs and Features and located the things to delete based on the PowerShell provided by Emmanuel Briney in the article I referenced.
